### PR TITLE
Toast component

### PR DESCRIPTION
**Pull request by lannisterrr**
Added - Custom Toast Component 
Review - First 7 files

### CODE REVIEW

File: src/Pages/Home.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { ListingVideoComponent } from '../components/ListingVideoComponent';
 import { useData } from '../contexts/data-context';
 
 function Home() {
+  console.log('render home');
```
Comment (junaidshaikh-js) [inline src/Pages/Home.js:8]: remove console logs

**Review by junaidshaikh-js — APPROVED** [review]
Good work! 

Clean code, good use of useImperativeHandle hook

File: src/components/Toast.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { useState, forwardRef, useImperativeHandle, useEffect } from 'react';
+
+const Toast = forwardRef((props, ref) => {
+  const [showToast, setShowToast] = useState(false);
+
+  useImperativeHandle(ref, () => ({
```
Comment (ashishnnnnn) [inline src/components/Toast.js:6]: This is nice ..

**Review by ashishnnnnn — APPROVED** [review]
Code seems pretty much good to merge ..

**Pull request merged**